test/dynamo/test_trace_rules.py

```diff
@@ -169,6 +174,27 @@ def generate_allow_list():
     torch_object_ids = dict()
     torch_objects = set()
 
+    def heuristic_record_if_in_graph_function(obj, module, name):
+        try:
+            if hasattr(obj, "__wrapped__") and obj is not torch.ops:
+                obj = obj.__wrapped__
+        except Exception:
```

Comment:
this try...except doesn't seem like it should be necessary

Reply:
CPU only platform loading CUDA functions would throw ```AttributeError```, I'll update the exception to ```AttributeError```.

Reply:
What exactly is obj here, in this case?

Reply:
My last comment is incorrect, I thought this is an exception at another place. This is used to record a decorated function, e.g., ```functools.lru_cache```. We should get the original function because the decorated object's type is not function.
try...except is not necessary any more, I'll remove it.

Reply:
The obj is any obj under torch, we will unwrap to get the original object. And the next few lines to check if this obj is a function/method type and record it. This is used to generate the in_graph_function list based on the heuristic.